kubernetes/cache/kube_cache_test.go

```diff
@@ -19,11 +19,15 @@ import (
 	"github.com/kiali/kiali/kubernetes/kubetest"
 )
 
-func newTestingKubeCache(t *testing.T, cfg *config.Config, objects ...runtime.Object) *kubeCache {
+const IstioAPIEnabled = true
+
+func newTestingKubeCache(t *testing.T, cfg *config.Config, istioAPIEnabled bool, objects ...runtime.Object) *kubeCache {
```

Comment:
It probably would've been better when I wrote this to accept a `k8s kubernetes.ClientInterface` here rather than the `objects` so that you could set some options for the fake client. You don't need to refactor but can you copy the setup code into that test that needs to edit the client rather than adding more opts to this setup func? Otherwise these things quickly grow without bound and then it's hard to use and isn't much of a "helper" method.